pkg/util/kubernetes/apiserver/apiserver_nocompile.go

```diff
@@ -61,3 +63,9 @@ func GetNodeLabels(_ *APIClient, nodeName string) (map[string]string, error) {
 	log.Errorf("GetNodeLabels not implemented %s", ErrNotCompiled.Error())
 	return nil, nil
 }
+
+// GetKubeClient returns a Kubernetes client.
+func GetKubeClient(timeout time.Duration) (kubernetes.Interface, error) {
+	log.Errorf("GetKubeClient not implemented %s", ErrNotCompiled.Error())
+	return nil, nil
```

Comment:
You're right. It makes no sense to return `nil, nil` here. Not sure why wrote that 🤦‍♂️ 

I think I got confused because all the methods in this file follow that pattern. We should probably change them, but I'd leave that for a future PR, because it's an unrelated change and I'm not sure what could break.